assets/scss/components/stencil/heroCarousel/_heroCarousel.scss

```diff
@@ -83,6 +85,12 @@
     background-size: cover;
     position: relative;
 
+    &.heroCarousel-slide--stretch {
```

Comment:
Thanks for doing the change! `heroCarousel-slide--stretch` doesn't need to be nested in `heroCarousel-slide`. Just move the former below the latter.